src/node.cc

```diff
@@ -1383,27 +1384,28 @@ void AppendExceptionLine(Environment* env,
                      linenum,
                      sourceline_string);
   CHECK_GE(off, 0);
```

Comment:
snprintf returns `-1` only on an encoding error else it will return the number of characters that would have been written if if n had been sufficiently large. In #2581 it triggers at https://github.com/nodejs/node/blob/master/src/node.cc#L1379